### PR TITLE
PLA-6918: Modified service paramet for rules to make it more specific

### PR DESCRIPTION
**Pull request by ajayshekar**

### CODE REVIEW

File: services/config.rb [hunk 1]
```diff
@@ -1025,7 +1025,7 @@
 
 coreo_aws_rule "azure-sql-threat-detection-on-database" do
   action :define
-  service :sql
+  service "Microsoft.Databases"
```
Comment (ddeep2007) [inline services/config.rb]: This should be "Microsoft.Sql"

File: services/config.rb [hunk 2]
```diff
@@ -1058,7 +1058,7 @@
 
 coreo_aws_rule "azure-sql-threat-detection-types-all-database" do
   action :define
-  service :sql
+  service "Microsoft.Databases"
```
Comment (ddeep2007) [inline services/config.rb]: All the rest as well need to be modified to "Microsoft.Sql"

File: services/config.rb [hunk 3]
```diff
@@ -1783,7 +1783,7 @@
 
 coreo_aws_rule "azure-monitoring-activity-log-alert-for-delete-network-security-solution" do
   action :define
-  service :monitoring
+  service "Microsoft.Insights"
```
Comment (ddeep2007) [inline services/config.rb]: Are these case sensitive?

File: services/config.rb [hunk 4]
```diff
@@ -1869,7 +1869,7 @@
 
 coreo_aws_rule "azure-security-ssh-access-restricted-from-internet" do
   action :define
-  service :security
+  service "Microsoft.Networking"
```
Comment (ddeep2007) [inline services/config.rb]: Should be Microsoft.Network. Correct the remaining as well.

File: services/config.rb [hunk 5]
```diff
@@ -1391,7 +1391,7 @@
 
 coreo_aws_rule "azure-key-vault-logging-for-keyvault-enabled" do
   action :define
-  service :key_vault
+  service "Microsoft.Security"
```
Comment (ddeep2007) [inline services/config.rb]: Should be Microsoft.KeyVault

File: services/config.rb (hunk 1 above)
Comment (ajayshekar) [reply services/config.rb]: Making the changes

File: services/config.rb (hunk 2 above)
Comment (ajayshekar) [reply services/config.rb]: Isn't SQL part of Microsoft.Databases service?

Comment (ddeep2007) [reply services/config.rb]: There isn't a Microsoft.Databases service. We are trying to map it to the actual Azure service names.

**Pull request merged**